packages/swirl-components/src/components/swirl-form-control/swirl-form-control.tsx

```diff
@@ -242,7 +243,15 @@ export class SwirlFormControl {
             <LabelTag class="form-control__label" onClick={this.onLabelClick}>
               <span class="form-control__label-text" id={this.labelId}>
                 {this.label}
+                {this.tooltip && this.labelPosition === "outside" && (
+                  <span class="form-control__tooltip">
+                    <swirl-tooltip content={this.tooltip} positioning="fixed">
```

Comment:
**issue:** 

i dont know how long the texts in the tooltip will get, but we should ensure it renders correctly. i just checked if the tooltip component itself is able to display a very long text
<img width="518" alt="image" src="https://github.com/getflip/swirl/assets/23700195/9d940d19-59f2-44b7-b730-d013a44c703b"> it does work. 

unfortunately the same or just a lengthy text gets cut in the component here. 
<img width="886" alt="image" src="https://github.com/getflip/swirl/assets/23700195/885256a6-640e-465e-8c96-832f4417e64c">

Reply:
Done